keps/sig-node/20190129-hugepages.md

```diff
@@ -270,6 +297,58 @@ spec:
       medium: HugePages
 ```
 
+This is an example of an invalid pod that requests hugepages of two
+differfent sizes, but doesn't use `medium: Hugepages-<size>` notation.
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+spec:
+  containers:
+...
+    volumeMounts:
+    - mountPath: /hugepages
+      name: hugepage
+    resources:
+      requests:
+        hugepages-2Mi: 1Gi
+        hugepages-1Gi: 2Gi
+      limits:
+        hugepages-2Mi: 1Gi
+        hugepages-1Gi: 2Gi
+  volumes:
+  - name: hugepage
+    emptyDir:
```

Comment:
What page size backs the volume?  It seems this should fail validation if more than one page size is defined.

Reply:
Yes, that's exactly the case. There is an example of the invalid pod. Probably I should add more explicit explanation. Will do.